README.md

```diff
@@ -12,9 +12,9 @@ run distributed or non-distributed TensorFlow/PyTorch/MXNet/XGBoost jobs on Kube
 
 - For a complete reference of the custom resource definitions, please refer to the API Definition.
   - [Tensorflow API Definition](pkg/apis/tensorflow/v1/types.go)
```

Comment:
We probably want to make these consistent. Seems like https://github.com/kubeflow/tf-operator/blob/master/pkg/apis/tensorflow/v1/types.go is still there.

Reply:
Are you suggesting that we change the name of the file?
I noticed that the file names are explicit for all (MXNet, Pytorch, and XGboost) except TensorFlow which has `types.go` instead of `tflow_types.go`?

Reply:
Perhaps changing those files to types.go would be good since they are already under each particular framework.

Reply:
Sounds good. I will make the changes in a new PR and link it to this one.

Reply:
Hmm why not reusing this same PR? I don't think the changes here are needed and will be reverted by the new PR anyways.

Reply:
Oh yes. that makes sense. I will update the change here itself. 👍🏽